src/api/telemetry/TelemetryAPI.js

```diff
@@ -131,8 +130,9 @@ export default class TelemetryAPI {
 
     /**
      * @private
+     * Though also used by TelemetryCollection
```

Comment:
In this case, should we remove the `@private`?

Reply:
We probably should 😀, I was just reluctant as @akhenry specifically [wanted it private](https://github.com/akhenry/openmct-yamcs/issues/139#issuecomment-1283271861).